flink-cdc-common/src/main/java/org/apache/flink/cdc/common/configuration/ConfigurationUtils.java

```diff
@@ -222,7 +222,7 @@ static Float convertToFloat(Object o) {
             } else {
                 throw new IllegalArgumentException(
                         String.format(
-                                "Configuration value %s overflows/underflows the float type.",
+                                "Configuration value %s overflows/underflow's the float type.",
```

Comment:
Seems `underflows` is correct as 3rd person singular, not possessive form.

Reply:
yes, I will revert it